src/batch-client/index.ts

```diff
@@ -432,6 +433,28 @@ export class ZimbraBatchClient {
 			content
 		}));
 
+	public enableTwoFactorAuth = (options: EnableTwoFactorAuthInput) =>
+		this.jsonRequest({
+			name: 'EnableTwoFactorAuth',
+			body: {
+				name: {
+					_content: options.name
+				},
+				password: options.password
+					? {
+							_content: options.password
+					  }
+					: null,
+				authToken: options.authToken,
+				twoFactorCode: options.twoFactorCode
```

Comment:
use destructuring to add optional elements in object

... twoFactorCode && {
     twoFactorCode
}

Reply:
done